locales/de.json

```diff
@@ -49,25 +79,275 @@
+				"invitationLink": "Einladungslink",
+				"pleaseNote": "Bitte beachte:",
+				"renameClass": "Klasse umbenennen",
+				"student": "Schüler",
+				"teacher": "Lehrer ",
+				"termOfUse": "Nutzungsordnung",
+				"youCanConsentToThisOnBehalfOfYourStudent": "Du kannst auf dieser Seite im Namen deiner Schüler das Einverständnis zur",
+				"yourStudentsAreNotYetInThe": "Deine Schüler sind noch nicht in der"
+			},
+			"text": {
+				"andTo": "und zur",
+				"attentionIfYouChangeTheSchoolYear": "Achtung! Wenn du das Schuljahr wechselst wird deine Auswahl an Vorjahres-Klassen zurückgesetzt.",
+				"byClickingOn": "Durch den Klick auf",
+				"consentConfirmation": "bestätigst du, dass du das Einverständnis der Schüler bzw. deren Eltern auf anderem Wege erhalten hast.",
+				"forAllStudents": "Für alle Schüler",
+				"inviteTheParentsViaLink": " Lade die Eltern per Link ein, ihre Kinder in deiner Klasse zu registrieren und die elektronische Einwilligungserklärung abzugeben.",
+				"manuallyGiveConsentForAllStudent": "Einverständnis manuell für alle Schüler erklären",
+				"nextPageYouWillReceiveConfirmation": "Auf der nächsten Seite erhältst du eine Bestätigung und hast die Möglichkeit, alle Startpasswörter zusammen mit einer Anleitung für deine Schüler auszudrucken.",
+				"noLackOfInformedConsent": "In dieser Klasse gibt es keine fehlenden Einverständniserklärungen.",
+				"ofThe": "der",
+				"required": "erforderlich",
+				"toExplain": "erklären.",
+				"withoutAFullDeclarationOfConsent": " ohne vollständige Einverständniserklärung:"
+			}
+		},
+		"global": {
+			"button": {
+				"addCourse": " Kurs hinzufügen",
+				"addNewAppoinment": "Neuen Termin hinzufügen",
+				"contactSupport": "Support kontaktieren",
+				"createLinkSheet": "Link-Druckbogen erstellen",
+				"filterAdd2": "Filter hinzufügen",
+				"filterApply2": "Übernehmen",
+				"sendLinksToStudentEmailAddresses": "Links an Schüler E-Mail Adressen senden"
+			},
+			"label": {
+				"afterSavingAllStudentsAdded": "(Nach dem Speichern werden alle Schüler automatisch hinzugefügt!)",
+				"chooseDate": "Termine auswählen",
+				"classes": "Klassen:",
+				"colorOfTheCourse": "Farbe des Kurses:",
+				"courseName": "Name des Kurses",
+				"dataProtection": "Datenschutz",
+				"dateOfBirth": "Geburtsdatum (optional):",
+				"emailAddress": "E-Mail-Adresse:",
+				"firstName": "Vorname:",
+				"from": "Von",
+				"lastName": "Nachname:",
+				"lengthOfLesson": "Dauer der Schulstunde",
+				"room": "Raum",
+				"startOfLesson": "Start der Schulstunde",
+				"studentParticipants": "Schüler / Teilnehmer",
+				"substituteTeacher": "Vertretungs Lehrer",
+				"teachingTeacher": "Unterrichtender Lehrer",
+				"termsOfUse": "Nutzungsbedingungen",
+				"timeSpan": "Zeitraum (z.B. Schuljahr)",
+				"to": "Bis",
+				"weekday": "Wochentag"
+			},
+			"text": {
+				"areYouSureYouWantToDeleteTheFollowingItem": "Sind Sie sich sicher, dass Sie folgendes Element löschen wollen?",
```

Comment:
please update everything to you, we could separate into a formal/informal german language but no mix.